Add contrib script that replicates vixie cron mail subject/from

isodude opened the pull request:
Sometimes it's ideal to replicate how vixie cron sends mail, to ease switching to systemd-cron.

Closes #166

nabijaczleweli commented:
You wrote this for... bash? I think? idk. I updated this to work under /bin/sh, handle `@daily whatever ...` Subjects, and, rather than taking an explicit `MAILFROM=user@hostname`, expanding it, and parsing it back, m_f_j just does `export user` so the interceptor can use it directly (see diff), which should be more consistent? I think I didn't break anything, but can you test this?

isodude commented:
I use bash daily and sh symlinks to bash I think. Hard to think about it in sh-terms.

The `export user` clearly is a much neater way to do it. I do think I tested it but must have failed.

I rebased also, need to force push with just two commits though.

I updated such that hostname is always the short one in i subject. Sometimes this might not be true with `hostname`.

isodude commented:
Do amend you name to the commits since it's your code, if you feel like it, or re-commit.

isodude commented:
It seems that when you pushed code ( I think ), to this PR, it changed the reference here https://github.com/systemd-cron/systemd-cron/issues/163#event-16338720482. I think we hit a github bug!

nabijaczleweli commented:
I think you merged from trunk which pulled in fa1fb16f2f567035243260ff6da325b409d7243c and github mis-detected this as you committing it somehow? definitely weird

Applied as 5fd96f1f4a61d13b6b001c8930e8df6c59761eb1, thanks!
